custom-elements/custom-elements-registry/define.html

```diff
@@ -0,0 +1,230 @@
+<!DOCTYPE html>
+<title>Custom Elements: Element definition</title>
+<link rel="help" href="https://html.spec.whatwg.org/multipage/scripting.html#element-definition">
+<meta name="author" title="Koji Ishii" href="mailto:kojiishi@gmail.com">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<body>
+<div id="log"></div>
+<script>
+'use strict';
+(function () {
+  var testable = false;
+  test(function () {
+    assert_true('customElements' in window, '"window.customElements" exists');
+    assert_true('define' in window.customElements, '"window.customElements.define" exists');
+    testable = true;
+  }, '"window.customElements.define" exists');
+  if (!testable)
+    return;
+
+  // 1. If IsConstructor(constructor) is false,
+  // then throw a TypeError and abort these steps.
+  test(function () {
+    assert_throws(new TypeError, function () {
+      window.customElements.define();
+    });
+  }, 'If no arguments, throw a TypeError');
```

Comment:
Can you elaborate? I didn't understand "make this brief, like a test name" part. Assertion message is omitted because only one assert in a test and the test has description, so it looked redundant to me, but maybe I misunderstood the whole sentence?